integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/TestLoadDataGeneral.scala

```diff
@@ -234,9 +234,25 @@ class TestLoadDataGeneral extends QueryTest with BeforeAndAfterEach {
       CarbonCommonConstants.BLOCKLET_SIZE_DEFAULT_VAL)
   }
 
+  test("test decimal value as null with global sort load") {
```

Comment:
@kunal642 had already fixed one issue regarding null value for string, now we got for decimal. He has added test case with string, int, double, int, bigint having null values in the test case, now decimal is added, please better to add a test case for all complex types also once.